VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,67 @@
+# Video 5: Open governance and transparency
+
+- AsyncAPI and Linux Foundation
+- What is transparency
+- What is open governance
+- Why AsyncAPI is committed to this model
+- Putting theory into practice
+
+Hello everyone! Welcome to this video on how AsyncAPI works. Today, we will talk about transparency, open governance, and why AsyncAPI is committed to this model. Do you want to learn more? Come over!
+
+## AsyncAPI and Linux Foundation
+
+In 2020, AsyncAPI started to grow exponentially and impacted the API world significantly. At that still initial moment, the project decides to associate with Postman. It was then that discussions began on what was the best way to improve governance within the project. 
+
+If one thing was crystal clear, it was that AsyncAPI should join a neutral home. But not just any neutral home. It needed to join a well-established and trusted organization. Housing the project on such a solid and neutral ground would help to develop its independence and community-driven management. 
+
+Convinced on the idea of democratizing power and devoted to establishing a horizontal model-based, AsyncAPI had the goal, from its very beginning, of fostering multiple points of view that could enrich its vision, decision-making, and governance. 
+
+With these ideas and principles clear, after a long time of searches and negotiations, AsyncAPI joined the Linux Foundation umbrella in March 2021. This move granted neutrality and transparency, ensuring that the specification will continue to be driven democratically by the people who contribute their time and effort to the project.
```

Comment:
```suggestion
With these ideas and principles clear, after a long time of searches and negotiations, AsyncAPI joined the Linux Foundation umbrella in March 2021. This move granted neutrality and open governance, ensuring that the specification will continue to be driven democratically by the people who contribute their time and effort to the project.
```

yeah, LF cannot guarantee transparency, it was up to us and the open governance rules we put into the project and that we respec on daily basis